data/crac/crac-util/src/main/java/com/farao_community/farao/data/crac_util/CracCleaningFeature.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright (c) 2020, RTE (http://www.rte-france.com)
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_util;
+
+/**
+ * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
+ */
+
+public enum CracCleaningFeature {
+    CHECK_CNEC_MNEC(true),
+    REMOVE_UNHANDLED_CONTINGENCIES(false);
+
+    boolean enabled;
```

Comment:
I'm not sure this is what @murgeyseb was thinking about, as the enabling/disabling are here hard-coded and cannot be changed by the user without changing the code

Reply:
You change it like this. Is it what you were thinking about?
        CracCleaner cracCleaner = new CracCleaner();
        cracCleaner.disableFeature(CracCleaningFeature.CHECK_CNEC_MNEC);
        cracCleaner.enableFeature(CracCleaningFeature.REMOVE_UNHANDLED_CONTINGENCIES);
        cracCleaner.cleanCrac(crac, network);